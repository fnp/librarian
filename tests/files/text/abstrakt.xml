<utwor>
  <opowiadanie>
    <abstrakt>
      <akap><tytul_dziela>Przygody Tomka Sawyera</tytul_dziela> to najsłynniejsza powieść dla młodzieży autorstwa amerykańskiego pisarza Marka Twaina.</akap>
    </abstrakt>
  </opowiadanie>
</utwor>
