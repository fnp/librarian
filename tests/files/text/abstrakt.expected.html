<blockquote>
      <p class="wl paragraph"><em class="book-title">Przygody Tomka Sawyera</em> to najsłynniejsza powieść dla młodzieży autorstwa amerykańskiego pisarza Marka Twaina.</p>
    </blockquote>
